stats/stats.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Stats.js r12
+// Type definitions for Stats.js r14
 // Project: http://github.com/mrdoob/stats.js
-// Definitions by: Gregory Dalton <https://github.com/gregolai>
```

Comment:
keep original author name. like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3